mission_planner2/UAVState[] other_UAVs_state